index.yaml

```diff
@@ -368,7 +346,7 @@ entries:
     name: nodered-chart
     type: application
     urls:
-    - https://github.com/rhm-samples/edge-helm-charts/blob/master/nodered-chart-0.0.2.tgz
+    - https://github.com/rhm-samples/edge-helm-charts/blob/master/nodered-chart-0.0.2.tgz?raw=true
```

Comment:
This change is correct, but does not match what the report for 0.0.2 version has, lets remove this change here and send another PR for 0.0.3 version with the new url in the report.